src/libstd/sys/wasm/condvar_atomics.rs

```diff
@@ -41,14 +41,26 @@ impl Condvar {
     }
 
     pub unsafe fn notify_one(&self) {
-        self.cnt.fetch_add(1, SeqCst);
-        wasm32::atomic_notify(self.ptr(), 1);
+        // SAFETY: The caller must gurantee that the argument of `fetch_add()` is valid
+        // to describe memory ordering.
+        unsafe {
+            self.cnt.fetch_add(1, SeqCst);
+        }
+        unsafe {
+            wasm32::atomic_notify(self.ptr(), 1);
```

Comment:
My guess is that the unsafety here is basically just from the raw pointer, so we can say something like "ptr() is always valid"